components/polls/PollForm.tsx

```diff
@@ -319,10 +376,104 @@ const PollForm: React.FC<BoxProps> = ({ ...props }) => {
               )}
             />
           </FormControl>
+
+          {/*  ██████╗ █████╗ ██╗      ██████╗██╗   ██╗██╗      █████╗ ████████╗██╗ ██████╗ ███╗   ██╗███████╗
+          {/* ██╔════╝██╔══██╗██║     ██╔════╝██║   ██║██║     ██╔══██╗╚══██╔══╝██║██╔═══██╗████╗  ██║██╔════╝
+          {/* ██║     ███████║██║     ██║     ██║   ██║██║     ███████║   ██║   ██║██║   ██║██╔██╗ ██║███████╗
+          {/* ██║     ██╔══██║██║     ██║     ██║   ██║██║     ██╔══██║   ██║   ██║██║   ██║██║╚██╗██║╚════██║
+          {/* ╚██████╗██║  ██║███████╗╚██████╗╚██████╔╝███████╗██║  ██║   ██║   ██║╚██████╔╝██║ ╚████║███████║
+          {/*  ╚═════╝╚═╝  ╚═╝╚══════╝ ╚═════╝ ╚═════╝ ╚══════╝╚═╝  ╚═╝   ╚═╝   ╚═╝ ╚═════╝ ╚═╝  ╚═══╝╚══════╝
+          {/*                                                                                                  */}
+          <VStack spacing={2} marginTop={6}>
+            <Heading fontSize="3xl" as="h1">Calculations</Heading>
+            <Text fontSize="m">How should votes be tallied?</Text>
+          </VStack>
+
+           <FormControl>
+            <FormLabel mt='1rem' htmlFor='tokenStrategies'>
+              Token Strategy
+            </FormLabel>
+            <Controller
+              control={control}
+              name='token_strategies'
+              {...(!getValues('token_strategies')) ? setValue('token_strategies', defaultStratId ? defaultStratId : '') : null}
+              render={({ field: { onBlur } }) => (
+                <Select
+                  id='tokenStrategies'
+                  options={strategies}
+                  defaultValue={{ label: EMPTY_STRATEGY_NAME, value: defaultStratId }}
+                    // isMulti
+                  isSearchable
+                  onBlur={onBlur}
+                  onChange={i => {
+                    console.log({ i })
+                    setValue(
+                      'token_strategies',
+                      i?.value ?? ''
+                    )
+                    clearErrors('token_strategies');
+                    (i?.label === EMPTY_STRATEGY_NAME) ? setIsTokenVote(false) : setIsTokenVote(true);
+                  }}
+                />
+              )}
+            />
+          </FormControl>
+
+          <FormControl isInvalid={!!errors.block_height?.message}>
```

Comment:
Hide this field if "Bankless Dao" strategy is not set (the logic should really be "if not token-based strategy".

Reply:
so we might need a "strategy type" prop in the future, for now we can hard-code